src/Log/Engine/FileLog.php

```diff
@@ -116,9 +116,9 @@ public function __construct(array $config = [])
      *    See Cake\Log\Log::$_levels for list of possible levels.
      * @param string $message The message you want to log.
      * @param array $context Additional information about the logged message
-     * @return bool success of write.
+     * @return void
      */
-    public function log($level, $message, array $context = []): bool
+    public function log($level, $message, array $context = [])
```

Comment:
Can we add `:void`?
Or will that break due to the different interface?

Reply:
The interface doesn't have return type hint so best to not add our own.